extensions/azurecore/src/account-provider/auths/azureAuth.ts

```diff
@@ -323,16 +321,17 @@ export abstract class AzureAuth implements vscode.Disposable {
 			Logger.error(`Error: Could not fetch the azure resource ${azureResource} `);
 			return null;
 		}
+		const endpoint = resource.endpoint;
 		let account: AccountInfo | null = await this.getAccountFromMsalCache(accountId);
 		if (!account) {
 			Logger.error('Error: Could not fetch account when acquiring token');
 			return null;
 		}
 		let newScope;
 		if (resource.azureResourceId === azdata.AzureResource.ResourceManagement) {
-			newScope = [`${resource?.endpoint}user_impersonation`];
+			newScope = [`${endpoint.endsWith('/') ? endpoint : endpoint + '/'}user_impersonation`];
```

Comment:
You could just do this on line 324 to save having to duplicate the call (and adding a comment about why this is necessary wouldn't hurt either)